contracts/bridge/BridgeFee.sol

```diff
@@ -19,15 +19,29 @@ contract BridgeFee {
         receiver = _receiver;
     }
 
-    function _payKLAYFee(uint256 _fee) internal {
-        if(isValidReceiver() == true && _fee > 0) {
-            receiver.transfer(_fee);
+    function _payKLAYFeeAndRefundChange(uint256 _feeLimit) internal {
+        uint256 fee = feeOfKLAY;
+
+        if(isValidReceiver() == true && fee > 0) {
+            require(_feeLimit >= fee, "invalid feeLimit");
+
+            receiver.transfer(fee);
+            msg.sender.transfer(_feeLimit.sub(fee));
```

Comment:
If _feeLimit == fee, this transfer send zero value. Is it O.K.?

Reply:
I considered checking zero value. But it can save a little gas.
For simplify code, I would like to stay this code. 
I will compare the used gas, later.